src/query/proposals.graphql

```diff
@@ -6,10 +6,50 @@ query proposals($first: int, $offset: int) {
     }
   }
   proposals(func: uid(proposals), orderdesc: created_date, first: $first, offset: $offset) {
```

Comment:
No need to modify this file, it's used for multi-tenancy and is not being used in production and I'm actively developing it right now